src/dashboard.js

```diff
@@ -7488,6 +7488,10 @@ let chartState = function (element, chartIndex) {
         if (this.dimensions) {
             this.data_url += "&dimensions=" + this.dimensions;
         }
+                
+        if (this.aggr_method) {
+            this.data_url += "&aggr_method=" + this.aggr_method;
+        }
```

Comment:
It doesn't look like it's intentional, is it a mistake? There is no `aggr_method` inside dashboard.js. To be clear, this file is used only for static custom dashboards in the Agent (like tv.html, etc.)

(same goes for `public/dashboard.js` file)

Reply:
I wanted to discuss about this. Removed and we can discuss it later on

Reply:
@jacekkolasa I wanted to ask also whether it makes sense for the aggregation methods to be set in the relevant `snapshot-loaders`

Reply:
Right now we don't need to have it there, but they can stay, so we don't miss it when we'll introduce aggr methods in Agent Dashboard